core/domain/email_manager.py

```diff
@@ -77,6 +77,46 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+_EDITOR_ROLE_EMAIL_SUBJECT = (
+    '%s added you to one of their explorations on Oppia.org')
+
+_EDITOR_ROLE_EMAIL_BODY = (
+    '%s added you to his exploration, \"%s\", as a %s.<br>'
```

Comment:
The inviter may not be male. I suggest:

```
%s has granted you %s permissions to their learning exploration, [link]"%s%[/link], on
Oppia.org. This allows you to:

    [bullet-point list]

You can access the exploration [link]here[/link]. Thanks!

Best wishes,
The Oppia Team
```

(Note that the two links point to the same place.)